README.md

```diff
@@ -12,22 +12,24 @@ This repository contains versioned TextField suggestion data for eq-questionnair
 | religions.json | List of religions |
 
 
-Source data files are provided by the business as multiple column csv files. These can be manually added/updated in this repository at ./source-data
+Source data files are provided by the business as multiple column csv files. These can be manually added/updated in this repository at `./source-data`
+
+Separate source data files are provided for Northern Ireland at `./source-data/ni`. The non Northern Ireland source data files are located at `./source-data/gb` 
 
 - source files to be provided by the business named for the above datasets
 
 - files to be UTF-8 csv
 
 - any values including commas should be double quoted
 
-- header row to contain comma separated language_codes (i.e `en_gb`, `cy`, `ga`, `eo`)
+- header row to contain comma separated language_codes (i.e `en-gb`, `cy`, `ga`, `eo`)
 
 - data rows to contain comma separated translated lookup values (as to be presented in the lookup lists)
 
 - null values for a given language item can be provided by a non-spaced commas. This will result in that term not being presented as a suggestion for that language
 
 json files can be generated from the source csv files using `./scripts/convert_csv_to_json.py`
 
-- expected source file location `./source-data`
+- default source file location `./source-data/gb`
 
-- json file output location `./data`
+- default json file output location `./data/gb`
```

Comment:
At the moment the script will only generate the `gb` files. The script should be updated to do both sets of files.

Reply:
Well my thoughts are its only a local dev script to aid the generation of the json files. It can be easily be updated to do any other sub folders as required.

Otherwise the options are to parameterise the sub folder to be processed (limiting it to accept either `gb` or `ni`) or to iterate over all files across all subfolders within `source-data`

Reply:
Updating the script to do all regions will save effort for every future run of the script. If we're not going to do that, then the Readme would need to be updated with instructions on how to edit the script, which seems counterproductive.

Reply:
The script should now deal with 'regional' sub directories